src/main.go

```diff
@@ -1,11 +1,35 @@
 package main
 
 import (
+	"bufio"
 	"flag"
+	"os"
 	"session"
+	"strings"
+	"chat"
 )
 
 func main() {
+
 	flag.Parse()
+
+	//kick off chat loop
+	go sendChatMessageLoop()
+
 	session.SetupSession()
 }
+
+
+func sendChatMessageLoop(){
+	for{
+		m:=getChatMessage()
```

Comment:
There are some egregious formatting imperfections around here. Please to run the following command:

gofmt -w src/\*.go src/\*/\*.go

It will automatically format your code.